src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+    background-repeat: no-repeat;
+  }
```

Comment:
all stars--N modifiers work as expected (Highlight first N stars)
 
 What if I change the modifier to 3 will it work?)

Reply:
Good notice) I'll changed it with SASS

Reply:
UPD: I've resolved that problem with scss, but then I realized I also have to change index.html accordingly to changes, but I can't make that without JS. It's so far to this JS topic, unforrtunately.
But I will be glad if you can give me a tip :D